src/xunit.performance.api/XunitPerformanceHarness.cs

```diff
@@ -1,88 +1,69 @@
+using MarkdownLog;
+using Microsoft.Xunit.Performance.Api.Table;
 using System;
 using System.Collections.Generic;
+using System.Globalization;
 using System.IO;
 using System.Linq;
-using System.Reflection;
-using Microsoft.Xunit.Performance;
-using Microsoft.Xunit.Performance.Api.Table;
+using System.Xml.Linq;
+using static Microsoft.Xunit.Performance.Api.XunitPerformanceLogger;
 
 namespace Microsoft.Xunit.Performance.Api
 {
     public sealed class XunitPerformanceHarness : IDisposable
     {
-        private string[] _args;
-
         public XunitPerformanceHarness(string[] args)
         {
             _args = args;
+            _disposed = false;
 
             // Set the run id.
-            Configuration.RunId = DateTimeOffset.UtcNow.ToString("yyyy-MM-dd-HH-mm-ss");
+            Configuration.RunId = $"{DateTimeOffset.UtcNow.ToString("yyyyMMddHHmmss")}";
```

Comment:
I can undo this change, but eventually this might be a parameter passed by the user like the runner. At least that is something @DrewScoggins mentioned, because it's easier to parse and do post processing on it.


---
In reply to: [100347473](https://github.com/Microsoft/xunit-performance/pull/183#discussion_r100347473) [](ancestors = 100347473)